tsl/src/continuous_aggs/materialize.c

```diff
@@ -633,6 +633,7 @@ create_materialization_plan(MaterializationContext *context, MaterializationPlan
 		char *query = materialization->create_statement(context);
 		Oid types[] = { context->materialization_range.type, context->materialization_range.type };
 
+		elog(DEBUG2, "%s: %s", __func__, query);
```

Comment:
I added a macro `TS_DEBUG_LOG` for these kinds of situations. It will be removed for release builds, so that we do not laden the server for debug printouts unnecessarily.

Reply:
Sometimes I need this DEBUG2 for debugging production materialization queries produced by our refresh cagg code (specially on SDCs).

Reply:
If it is needed for debugging production code, then `TS_DEBUG_LOG` should not be used. Agree.